contracts/core/25-handler/IBCUtil.sol

```diff
@@ -0,0 +1,23 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity ^0.8.9;
+
+library IBCUtil {
+    function process_delgatecall(
```

Comment:
This name is confusing, we are actually handling the result of a delegate-call right?
Suggest changing the name to `check_return`.

Reply:
fine, good suggestion